### PR TITLE
fix build sample in vc2010

### LINKED ISSUE #6264: error on build samples in vs2010
- OpenCV version: 3.x
- Windows XP x86
CMake: GCC 3.4.3

Error on build samples:
"Q:\ocv\opencv\build_x86_pre\OpenCV.sln" (целевой объект по умолчанию) (1) ->
"Q:\ocv\opencv\build_x86_pre\ALL_BUILD.vcxproj.metaproj" (целевой объект по умо
лчанию) (2) ->
"Q:\ocv\opencv\build_x86_pre\samples\cpp\example_detect_mser.vcxproj.metaproj"
(целевой объект по умолчанию) (86) ->
"Q:\ocv\opencv\build_x86_pre\samples\cpp\example_detect_mser.vcxproj" (целевой
объект по умолчанию) (87) ->
(Целевой объект ClCompile) ->
  ..\..\..\samples\cpp\detect_mser.cpp(285): error C2668: 'acos' : ambiguous ca
ll to overloaded function [Q:\ocv\opencv\build_x86_pre\samples\cpp\example_dete
ct_mser.vcxproj]

After replaceent
`float	pi = (float)acos(-1);` 
in detect_mser.cpp(285) to
`float	pi = (float)acos(static_cast<float>(-1));`
the build was successful.

### PR DESCRIPTION
**Pull request by Bleach665**
resolves #6264 

It fix bug on build opencv example AKAZE_tracking in vs 2010

### CODE REVIEW

**Comment by Bleach665** [comment]
PS. Tell me please if I'm doing something wrong. I'm newbie to teamwork on github.

**Comment by alalek** [comment]
Just click on ["Details"](http://pullrequest.opencv.org/#/summary/opencv) link above. Current failure is sporadic (random) and not related to your patch.
Failed build is restarted.

File: samples/cpp/detect_mser.cpp [hunk 1]
```diff
@@ -187,7 +187,7 @@ static void onMouse(int event, int x, int y, int flags, void*)
         else
             rObs -= (float)0.1;
     }
-    float pi = (float)acos(-1.0);
+    float	pi = static_cast<float>(CV_PI);
```
Comment (alalek) [inline samples/cpp/detect_mser.cpp]: Please don't use "tabs"

**Comment by Bleach665** [comment]
This check build is also sporadic?

**Comment by alalek** [comment]
:+1:

**Pull request merged**